pkg/api/validation/validation.go

```diff
@@ -2844,10 +2799,16 @@ func ValidateNodeUpdate(node, oldNode *api.Node) field.ErrorList {
 	// Clear status
 	oldNode.Status = node.Status
 
+	// update taints
```

Comment:
@davidopp -- i am not sure my thought was well-reasoned.  i was more trying to see if the problem space can be reduced.  the use-case of adding to an existing pod spec, and those specs getting out of date with replica sets still felt skewed.